app/views/order_items/_order_item.text.erb

```diff
@@ -0,0 +1 @@
+<%= sprintf("%d x %s", order_item.quantity, order_item.menu_item.description) %>
```

Comment:
Sorry for the trouble, but we are coding in Ruby and this looks like and even is valid C code. I think `"#{order_item.quantity} x #{order_item.menu_item.description}"` would be more Ruby-like.